sandbox/grommet-app/src/pages/layouts/kinds/RecordDetail/JobList.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  List,
+  NameValueList,
+  NameValuePair,
+  Text
+} from 'grommet';
+import { Edit, Trash } from 'grommet-icons';
+import { sentenceCase } from "../../../../utils/format";
+
+export const JobList = (
+  { jobs }:
+    {
+      jobs: {
+        id: string,
+        name: string,
+        [key: string]: any,
+      }[]
+    }
+) => {
+  return (
+    <List
+      data={jobs}
+      defaultItemProps={{ pad: { horizontal: 'none', vertical: 'xsmall' } }}
+    >
+      {(datum) => (
+        <Box
+          key={datum.id}
+          background="background-contrast"
+          round="small"
+          pad={{ left: 'small', right: 'xsmall', vertical: 'small' }}
```

Comment:
Would "small" all around work?

Reply:
Because we have an icon only button on the right, the padding in the button + pad on the container end up making the left-right whitespace feel unbalanced. Reducing the right pad to xsmall feels more balanced.

Reply:
I agree locally I changed it to `small` to be all around the same size but it felt off a little so I think even though the `right` is not the same visually looks better